tests/test_proxy_auxiliary.py

```diff
@@ -175,49 +201,44 @@ def test_check_compatibility_exception(mocker, cchannel_inst):
         proxy_inst = ProxyAuxiliary(cchannel_inst, [*AUX_LIST_INCOMPATIBLE])
 
 
-def test_create_auxiliary_instance_valid(mocker, cchannel_inst):
-    mocker.patch.object(ProxyAuxiliary, "run")
-    mocker.patch.object(CChannel, "open")
-
-    proxy_inst = ProxyAuxiliary(cchannel_inst, [*AUX_LIST_NAMES])
-    state = proxy_inst._create_auxiliary_instance()
-
-    assert state is True
-    assert proxy_inst.stop_event.is_set() is False
-
-
-def test_create_auxiliary_instance_exception(mocker, cchannel_inst):
+@pytest.mark.parametrize(
+    "side_effect_value, stop_event_set, expected_function_return",
+    [
+        (None, False, True),
+        (ValueError(), True, False),
+    ],
+)
+def test_create_auxiliary_instance_valid(
+    mocker, cchannel_inst, side_effect_value, stop_event_set, expected_function_return
+):
     mocker.patch.object(ProxyAuxiliary, "run")
-    mocker.patch.object(CChannel, "open", side_effect=ValueError())
+    mocker.patch.object(CChannel, "open", side_effect=side_effect_value)
```

Comment:
name could give more context - for example
```suggestion
    mocker.patch.object(CChannel, "open", side_effect=open_return_value)
```